### DISCUSSION TITLE
Choose timescaledb version

### DISCUSSION THREAD

**Discussion by brunodangelos** [autobase-tech/autobase#493] [Ideas]
It clould be possible to choose timescaledb version? Maybe in vars like the others applications.

**Comment by vitabaks** [comment]
The timescaledb package that is available in the timescale repository is being installed.

As a rule, several versions are available to create an extension, so you can choose the version that suits you the most.

**Reply by vitabaks** [reply]
lists the specific extension versions that are available for installation:

`select * from pg_available_extension_versions where name = 'timescaledb' order by version desc;
`

Install specific version:

`CREATE EXTENSION timescaledb WITH VERSION '2.11.2';
`

Update to specific version:

`ALTER EXTENSION timescaledb UPDATE TO '2.12.0';`
